app/views/job_invocations/_card_target_hosts.html.erb

```diff
@@ -0,0 +1,33 @@
+<% template_invocations = @job_invocation.pattern_template_invocations %>
+<div class="card-pf card-pf-accented">
```

Comment:
We'll need cards on other places soon too (auth sources page) and we also have some cards on smart proxy page. I wonder if it would be a good time to add some helpers to core. patternfly-react does not have cards component yet, Dominik might look into contributing this there but it will take some time. But if you think it's too much work, I'm fine with having card tags here :-)

Reply:
I would suggest trying to contribute cards regardless, it should not be a huge effort imho just to add the html structure to patternfly-react.